### PR TITLE
Add initial lua files.

### PR DESCRIPTION
**Pull request by wrefgtzweve**
Adds the initial client lua file.

### CODE REVIEW

File: lua/autorun/client/cl_cfc_help.lua [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    else
+        window:SetSize( 640, 480 )
+    end
+
+    window:Center()
+    window:SetTitle( "CFC Help" )
+    window:MakePopup()
+
+    window.Paint = function( self, w, h )
+    draw.RoundedBox( 8, 0, 0, w, h, Color( 36, 41, 67, 255 ) )
```
Comment (plally) [inline lua/autorun/client/cl_cfc_help.lua:15]: indent this and perhaps put this color in a constant at the top of the file

File: lua/autorun/client/cl_cfc_help.lua [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    end
+
+    local html = vgui.Create( "DHTML", window )
+
+    html:Dock( FILL )
+    html:OpenURL("https://cfcservers.org/learn/")
+end
+
+local function hideScrollbar ()
```
Comment (plally) [inline lua/autorun/client/cl_cfc_help.lua:24]:
```suggestion
local function hideScrollbar()
```

File: lua/autorun/client/cl_cfc_help.lua [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+    -- WIP
+end
+
+concommand.Add( "cfc_help", openHelp )
+
+hook.Add( "OnPlayerChat", "CFC_Help_OpenHelpCommand", function( ply, msg )
+
```
Comment (plally) [inline lua/autorun/client/cl_cfc_help.lua:31]:
```suggestion
```

File: lua/autorun/client/cl_cfc_help.lua [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    if string.sub( msg, 1, 5 ) ~= "!help" then return end
```
Comment (plally) [inline lua/autorun/client/cl_cfc_help.lua:32]: https://wiki.facepunch.com/gmod/string.StartWith
using StartsWith will make it easier to change this later

File: lua/autorun/client/cl_cfc_help.lua [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    window:MakePopup()
+
+    window.Paint = function( self, w, h )
+    draw.RoundedBox( 8, 0, 0, w, h, Color( 36, 41, 67, 255 ) )
+    end
+
+    local html = vgui.Create( "DHTML", window )
+
+    html:Dock( FILL )
+    html:OpenURL("https://cfcservers.org/learn/")
```
Comment (plally) [inline lua/autorun/client/cl_cfc_help.lua:21]:
```suggestion
    html:OpenURL( "https://cfcservers.org/learn/" )
```

**Review by plally — CHANGES_REQUESTED** [review]

**Review by brandonsturgeon — APPROVED** [review]

**Review by plally — APPROVED** [review]

**Pull request merged**